packages/console/src/pages/Users/components/CreateForm/index.tsx

```diff
@@ -44,7 +44,6 @@ const CreateForm = ({ onClose }: Props) => {
   return (
     <ModalLayout
       title="users.create"
-      subtitle="users.subtitle"
```

Comment:
should remove the related description in en.ts & zh-cn.ts

Reply:
it's used for user mgmt page

Reply:
No, we still need this subtitle in `Users` page